src/input.ts

```diff
@@ -3,21 +3,29 @@ import fs from "fs";
 import path from "path";
 import type { Collection } from "./types";
 
-interface YamlInput {
+export interface NetlifyCMSConfig {
```

Comment:
`NetlifyCMSConfig` type could be moved to the `types` directory, as it is referenced outside this file as well, either into its own file or appended to `types/fields.ts`.